evansinho/Weather-App
# Weather

## What Does This PR Do?
#### this PR contains a working Weather- application, fulfilling the requirements gotten from [here](https://www.theodinproject.com/courses/javascript/lessons/weather-app)

##  How Can It Be Tested?
- Visit [this](https://raw.githack.com/evansinho/Weather-App/weather/dist/index.html) page.
- Click on the location a model pops up.
- Which prompts you to input a location.
- Then the weather forecast and other info about the location is shown.
